README.md

```diff
@@ -378,6 +378,27 @@ After starting image service, users can open it via http://<image-server-ip>:707
 
 Unit test coverage report can be found at ./coverage
 
+### Run smoke test with FIT
+```
+WORKSPACE=`pwd`
+git clone https://github.com/RackHD/image-service.git
```

Comment:
Could we add a little explaination before git clone, like this:
1. Get latest image-service code and FIT code from git hub